docs/src/main/sphinx/sql/explain.rst

```diff
@@ -49,7 +49,7 @@ Examples
 EXPLAIN (TYPE LOGICAL)
 ^^^^^^^^^^^^^^^^^^^^^^
 
-Logical plan::
+Process the supplied query statement and create a logical plan in text format::
```

Comment:
Do we need to say "Process the supplied query statement" explicitly everywhere ? It seems obvious that the input query will be processed to create a plan from it.
Maybe we can put this in a less formal way "Create the logical plan in text format for a given query statement"

Reply:
Its about the same length .. and your suggestion is actually more format .. "for a given query statement" is very much like a math book.. I will leave as is